### ISSUE TITLE
Council update channel ownership

### ISSUE THREAD

**Issue by bedeho** [Joystream/joystream#3588]
# Background
Ultimately, creators will create channels through _applications_, not directly by sending raw transactions. This means these applications are at liberty to maliciously assign ownership of the channel to themselves, even if the creator is using an external wallet/signer to a browser DApp. An unsophisticated creator may not even notice this for a long time, until the channel has accumulated a substantial amount of value. For memberships this is much less of a real problem, both because members will on average be much more sophisticated, but also because payoff to taking over a membership is lower, as it does not directly hold value in the same way a as a channel, which has an audience and a channel account, NFTs and so on.

# Proposal
Introduce a proposal which allows channel to change ownership of a channel. This can be made quite safe by having a high threshold, quorum and constitutionality. Now, it will still be possible for the malicious application to drain the channel of substantial value if they detect such a proposal, however, at least the channel audience, content and creator token, can be reassigned to the correct owner.

This solution is not without it's flaws, but the alternative seems even worse.

Any ideas?

kdembler Lezek123 @ignazio-bovo

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202132419573087/1202209886670850) by [Unito](https://www.unito.io)

**Comment by kdembler** [comment]
User signing opaque transactions is a very general problem, not restricted to channels or ownership. Think about the dApp maliciously offering an NFT to an attacker for 1 JOY instead of putting it on sale. We could add this for channels, but what about memberships that can also hold significant value via NFTs? What about malicious NFT transfer? Where do we draw the line?

I see the value of council acting as fail-safe, kind of social recovery mechanism, but I think it creates a potentially dangerous precedent. I truly value self-custodial aspect of crypto, the promise that if I don't f-ck up, nobody can take my value/ownership from me. What if there will be some political disputes in the DAO and council would use this power to "seize" a channel, for whatever reason? I think this could be too much power in their hands

At the end of the day, I think this issue can only be really solved by sophisticated signer applications that will be able to double-check or at least present you detailed information on what you're signing.

**Comment by bedeho** [comment]
> I see the value of council acting as fail-safe, kind of social recovery mechanism, but I think it creates a potentially dangerous precedent. 

Yeah I agree. Not sure what to do here, just leaving it open for now.
